test/util/bitcoin-util-test.py

```diff
@@ -20,40 +20,44 @@
 import subprocess
 import sys
 
+
```

Comment:
For short util modules like this, and even for most of our test scripts, I don't think 2 lines separating each function is necessary. It makes sense for large projects where you want visual separation between top-level classes, but I think it's unnecessary here.